kernel_crawler/aliyunlinux.py

```diff
@@ -0,0 +1,44 @@
+from . import repo
+from . import rpm
+
+def v2_only(ver):
+    return ver.startswith('2')
+
+def v3_only(ver):
+    return ver.startswith('3')
+
+class AliyunLinux2Mirror(repo.Distro):
+    def __init__(self, arch):
+        mirrors = [
+            # AliyunLinux 2
+            # Mirror list on cloud-init config example:
+            # https://www.alibabacloud.com/help/en/elastic-compute-service/latest/use-alibaba-cloud-linux-2-images-in-an-on-premises-environment
+            rpm.RpmMirror('http://mirrors.aliyun.com/alinux/', 'os/' + arch + '/', v2_only),
+            rpm.RpmMirror('http://mirrors.aliyun.com/alinux/', 'updates/' + arch + '/', v2_only),
+            rpm.RpmMirror('http://mirrors.aliyun.com/alinux/', 'plus/' + arch + '/', v2_only),
+
+        ]
+        super(AliyunLinux2Mirror, self).__init__(mirrors, arch)
+
+    def to_driverkit_config(self, release, deps):
+        for dep in deps:
+            if dep.find("devel") != -1:
+                return repo.DriverKitConfig(release, "aliyunlinux2", dep)
```

Comment:
Is `aliyunlinux2` the os ID taken from `/etc/os-release`?

Reply:
on os-release it's `alinux` for both versions

Reply:
So, can we merge both into a single distro and call the `DriverkitConfig`  `alinux`?